code/game/machinery/aicore_lockdown.dm

```diff
@@ -0,0 +1,113 @@
+/obj/structure/machinery/aicore_lockdown
+	name = "AI Core Lockdown"
+	icon_state = "big_red_button_tablev"
+	unslashable = TRUE
+	unacidable = TRUE
+
+/obj/structure/machinery/aicore_lockdown/ex_act(severity)
+	return FALSE
+
+/obj/structure/machinery/aicore_lockdown/attack_remote(mob/user as mob)
+	return FALSE
+
+/obj/structure/machinery/aicore_lockdown/attack_alien(mob/user as mob)
+	return FALSE
+
+/obj/structure/machinery/aicore_lockdown/attackby(obj/item/attacking_item, mob/user)
+	return attack_hand(user)
+
+/obj/structure/machinery/aicore_lockdown/attack_hand(mob/living/user)
+	if(isxeno(user))
+		return FALSE
+	if(!allowed(user))
+		to_chat(user, SPAN_DANGER("Access Denied"))
+		flick(initial(icon_state) + "-denied", src)
+		return FALSE
+
+	if(!COOLDOWN_FINISHED(GLOB.ares_datacore, aicore_lockdown))
+		to_chat(user, SPAN_BOLDWARNING("AI Core Lockdown procedures are on cooldown! They will be ready in [COOLDOWN_SECONDSLEFT(GLOB.ares_datacore, aicore_lockdown)] seconds!"))
+		return FALSE
+
+	add_fingerprint(user)
+	aicore_lockdown(user)
+	COOLDOWN_START(GLOB.ares_datacore, aicore_lockdown, 2 MINUTES)
```

Comment:
This should be inside `aicore_lockdown` otherwise someone can quickly release a admin set lockdown. The cooldown only matters if you check `COOLDOWN_FINISHED` anyways, so it wouldn't prevent admins.